### ISSUE TITLE
Inputs registered twice

### ISSUE THREAD

**Issue by TheGershon** [libretro/fmsx-libretro#47]
Regardless of RA controller settings, inputs are registered twice. Happens either with emulated joystick or keyboard controls. I noticed this while entering a name in New Devil Golvellius, as it makes it impossible to enter anything coherent. Happens in the 3DS port as well, so it's not just an issue on RA for Win64.

![New Devil Golvellius (Disk 2)  EN -190704-030718](https://user-images.githubusercontent.com/6880780/60658848-e75bf080-9e08-11e9-96bc-d15da4cfdc0a.png)

**Comment by gingerbeardman** [comment]
If you can help me even get the game running (what settings?) then I can assist.

I downloaded it here: http://www.smspower.org/golvellius/golv2.html

**Comment by TheGershon** [comment]
That download didn't even load into the game for me, instead throwing me into the MSX prompt.
I got this one (in english) here: https://www.generation-msx.nl/software/compile/new-devil-golvellius/translations/1138/
As a side note, it seems the game requires swapping to the 3rd disk in order to save. Setting up an m3u file for the blueMSX core was easy, but this seems to be unsupported by fmsx. blueMSX also does not have the input problem. I'd use it if it had a 3DS port

**Comment by gingerbeardman** [comment]
Yeah, a port of blueMSX would be ideal but sadly it's not forthcoming. Meanwhile I try to fix or add what I can to fMSX.

**Comment by gingerbeardman** [comment]
I get the MSX prompt in both the above versions of the game... how odd. 

Will need to check more of my setup, system ROMs, etc

**Comment by TheGershon** [comment]
I had the same problem. Started launching other games (.rom) and tried it
again and it worked. Didn't change anything else so idk what I did exactly
to make it work.

**Comment by gingerbeardman** [comment]
I will try to find another game with keyboard entry that will work.

**Comment by cayce-msx** [comment]
I've been able to reproduce this one. I guess it has to do with the way the libretro.c wrapper translates retroarch keyboard event to the fmsx core; it does not happen with fMSX standalone. I'll see if I can fix it. For the moment, other issues have higher prio for me.

**Comment by cayce-msx** [comment]
Apologies, people. I've fiddled with this quite a bit but can't seem to find a proper fix. I do notice that this game has input quirks on other emulators as well, but no inputs being registered twice elsewhere. I'm hoping this is the only game having this problem on libretro-fmsx (it seems to implement its own keyboard scanning). For now, use blueMSX-libretro when playing this game...
